files/discords.json

```diff
@@ -15,6 +15,14 @@
         "fancyname": "Prism Launcher",
         "nicknames": ["prism community"]
     },
+    {
+        "id": "quickjoin",
+        "code": "Vp6Q4FHCzf",
+        "icon": "https://cdn.glitch.global/d204ee01-249d-4fd8-8b05-bea32f8a9bfa/standard%20(3).png",
```

Comment:
icons should be stored in the repo itself, see https://github.com/SkyblockClient/SkyblockClient-REPO/tree/main/files/icons